src/components/cell.js

```diff
@@ -33,6 +132,27 @@ export const ListSeparator = () => {
 };
 
 const styles = StyleSheet.create({
+  deviceId: {
+    backgroundColor: COLORS.gray4,
+    flex: 0.8,
+  },
+  secondaryDropdown: {
+    alignItems: 'center',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    backgroundColor: COLORS.gray4,
+    padding: 16,
```

Comment:
```suggestion
    padding: SPACING.small,
```
Isso vale para os outros de baixo tbm